tools/rush-update/src/rush.ts

```diff
@@ -22,10 +23,16 @@ export const getProjectFolders = once(() => getRushConfig()
   .then(({projects}) => projects.map((p) => p.projectFolder))
 );
 
-export async function updateRushShrinkwrapFile() {
+export async function updateRushShrinkwrapFile(): Promise<void> {
```

Comment:
Does this function have to be async?

Reply:
no `async` function has to be an async function, every `const x = await foo()` can be replaced with `foo().then(x => {...})`

this one `await`s 2 times